README.adoc

```diff
@@ -3,154 +3,63 @@
 :source-highlighter: prettify
 :project_id: gs-spring-boot-for-azure
 
-This article walks you through deploying a Spring Boot application to https://learn.microsoft.com/azure/spring-apps/overview[Azure Spring Apps].
+A Spring Boot hello-world application that includes everything you need to build and deploy an Azure solution. This application uses the Azure Developer CLI (azd) to get you up and running on Azure quickly. It includes application code and tools that serve as a foundation from which you can build upon and customize when creating your own solutions.
```

Comment:
I think the README modification will effect this guide: https://spring.io/guides/gs/spring-boot-for-azure/

Reply:
@fangjian0423 Then should we add a new section introducing AZD instead of just replace the current content?

Reply:
You can add a brief description of AZD if you think it will help people. Bear in mind that the guides are supposed to be 15-minute chunks of content, so please keep it short. Also, it's fair to link to Microsoft's AZD documentation in that brief description (I think that would be one good reason to add it).

Reply:
@Buzzardo thanks for the suggestion. We had planned to replace the current README completely with only introduction and instructions about how to run with AZD, just like what the PR has, and of course we will make sure the content could be read and run within 15 minutes. Do you think it is acceptable?